packages/client-cli/lib/frontend-openapi-generator.mjs

```diff
@@ -134,59 +134,41 @@ function generateFrontendImplementationFromOpenAPI ({ schema, name, language, fu
         writer.blankLine()
       }
 
-      if (method === 'get') {
-        let commonString = `const response = await fetch(\`\${url}${stringLiteralPath}`
+      writer.write('const headers =').block(() => {
+        writer.writeLine('\'Content-type\': \'application/json\'')
```

Comment:
Please include the charset. Unfortunately some obscure Chinese/Japanese system requires it.